server/lib/auth.js

```diff
@@ -93,16 +95,53 @@ export function verifyTwoFactorAuthenticatorCode(encryptedTwoFactorAuthToken, tw
   return verified;
 }
 
-export function enforceTwoFactorAuthenticationOnPayouts(req, twoFactorAuthenticatorCode) {
+const getCacheKey = userId => {
```

Comment:
If it's used only in one place, maybe we can just construct the key without having a function to do it.